src/include/mpir_handlemem.h

```diff
@@ -386,6 +431,18 @@ static inline void MPIR_Handle_obj_free_unsafe(MPIR_Object_alloc_t * objmem, voi
 
     obj->next = objmem->avail;
     objmem->avail = obj;
+    objmem->num_freed++;
+
+    if (is_info) {
+        /* This option slows down this routine.  This active garbage collection
```

Comment:
`is_info` is always called with constant and the functions are always inlined, so the compiler should optimize the branch away, so no slow down, right?

Reply:
Yes.